routes/class.js

```diff
@@ -24,4 +24,31 @@ exports.create = function(req, res){
 
 		console.log(classObject);
 	});
+};
+
+exports.subscribe = function(req, res){
+    if(!req.studentId) {
+        return resError(res, "Sorry, unable to find this user")
+    }
+    else if (!req.classId) {
+        return resError(res, "Sorry, unable to find this class")
+    }
+    
+    // Add Student._id to Class.studentIds
+    Class.findOne({ _id:req.classId }, function (err, class) {
+        if (err) return console.error(err);
+        
+        class.studentIds.push(res.studentId);
+        class.save(function(err){
+            if (err) return console.error(err);
+        });
+    });
+    
+    // Add the Class._id to Student.classes
+    Student.findOne({ _id:req.studentId }, function (err, student) {
```

Comment:
I think this can function can placed in the callback of `class.save()` above after the error checking, since that'll give us the freedom to only add the class to the student if the student was successfully added to the class.